doc/api/process.md

```diff
@@ -2305,6 +2305,222 @@ import { loadEnvFile } from 'node:process';
+   const process = require('node:process');
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+6. **Multiline values**:
+
+   * Values enclosed in double, single, or backtick quotes that span multiple
+     lines are accepted and stored with newline characters.
+
+   ```bash
+    MULTI_DOUBLE_QUOTED="double
+    quoted"
+
+    MULTI_SINGLE_QUOTED='single
+    quoted'
+
+    MULTI_BACKTICKED=`this
+    "multiline"
+    value`
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\nquoted');
+   assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'single\nquoted');
+   assert.strictEqual(process.env.MULTI_BACKTICKED,
+                      'this\n"multiline"\nvalue');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\nquoted');
+   assert.strictEqual(process.env.MULTI_SINGLE_QUOTED, 'single\nquoted');
+   assert.strictEqual(process.env.MULTI_BACKTICKED,
+                      'this\n"multiline"\nvalue');
+   ```
+
+7. **Escapes**:
+
+   * Newlines in double-quoted values are expanded to newline characters.
+   * Other escapes (e.g., `\n`) are treated as literal text in single-quoted
+     or unquoted values.
+
+   ```bash
+     EXPAND_NEWLINES="expand\nnew\nlines"
+     DONT_EXPAND_UNQUOTED=dontexpand\nnewlines
+     DONT_EXPAND_SQUOTED='dontexpand\nnewlines'
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.EXPAND_NEWLINES, 'expand\nnew\nlines');
+   assert.strictEqual(process.env.DONT_EXPAND_UNQUOTED,
+                      'dontexpand\\nnewlines');
+   assert.strictEqual(process.env.DONT_EXPAND_SQUOTED,
+                      'dontexpand\\nnewlines');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.EXPAND_NEWLINES, 'expand\nnew\nlines');
+   assert.strictEqual(process.env.DONT_EXPAND_UNQUOTED,
+                      'dontexpand\\nnewlines');
+   assert.strictEqual(process.env.DONT_EXPAND_SQUOTED,
+                      'dontexpand\\nnewlines');
+   ```
+
+8. **Export statements**:
+
+   * Any `export` keyword before a key is ignored, allowing compatibility with
+     shell scripts.
```

Comment:
I initially read this as saying that any lines beginning with `export` get ignored, not that just the `export` keyword alone is ignored.

```suggestion
   * Any `export` keyword before a key is ignored, allowing compatibility with
     shell scripts. The line is parsed as if `export` wasn't there.
```